app/src/main/res/values/strings.xml

```diff
@@ -18,16 +18,16 @@
     <string name="search_label">Search Publications</string>
     <string name="pf_placeholder">Publications Fragment</string>
     <string name="hello_blank_fragment">Hello blank fragment</string>
-    <string name="query_hint">Search by title...</string>
+    <string name="query_hint">Search by mTitle...</string>
```

Comment:
Accidental refactoring happened here. Supposed to stay 'title' but changed t mTitle.